docs/modules/operation/pages/deep-dive/admin/webui/opsboard/criteria-builder.adoc

```diff
@@ -17,130 +18,133 @@ It is possible to combine multiple criteria to display just a subset of informat
 
 
 | Asc
-| ascending order
+| Ascending order
 | -
 | -
 | -
 
 | Desc
-| descending order
+| Descending order
 | -
 | -
 | -
 
 | Between
-| Subset of data between value 1 and value 2
-| database attribute
+| Subset of data between value 1 and value 2.
+| Database attribute
 | String
 | String
 
 | Contains
-| Select all data that contains a given text string in a given database attribute.
-| database attribute
+| A text string to be compared against a database attribute.
+Any data containing the defined text string is selected.
+| Database attribute
 | String
 | -
 
 | Distinct
 | Select a single instance.
-| database attribute
+| Database attribute
 | -
 | -
 
 | Eq
-| database attribute
-| Select data where attribute equals (`==`) a given text string.
+| Select data where the defined attribute equals (`==`) a given text string.
+| Database attribute
 | String
 | -
 
 | Ge
-| Select data where attribute is greater or equal to (`>=`) a given text value.
-| database attribute
+| Select data where the defined attribute is greater or equal to (`>=`) a given text value.
+| Database attribute
 | String
 | -
 
 | Gt
-| Select data where attribute is greater than (`>`) a given text value.
-| database attribute
+| Select data where the defined attribute is greater than (`>`) a given text value.
+| Database attribute
 | String
 | -
 
 | Ilike
 | unknown
```

Comment:
Right? I'd love if @mmahacek or @uquevedo knew what these were!

Reply:
Using these two ilike and in within PostgreSQL, the ilike behaves similarly to the like restriction, but is case insensitive...which if these are IP addresses, might not matter...?

The in usually precedes a listing of known data to query against to filter down results.

A query in PostgreSQL would be: select * from events where nodeid in (54,55);

Only events for nodeids 54 and 55 would be returned.

@mmahacek might be able to give some better clarification since I'm only currently familiar with the PostgreSQL side of things.

Reply:
I dug into the code and `ilike` is a case incentive `like`.
```suggestion
| Select data where the defined attribute is like a given text value.
This option is similar to the SQL `ilike` function and is not case sensitive.
```